Formula/mg.rb

```diff
@@ -1,10 +1,8 @@
 class Mg < Formula
-  desc "Small Emacs-like editor"
-  # https://devio.us/~bcallah/mg/ is temporarily offline
+  desc "Small Emacs-like editor from OpenBSD"
   homepage "https://github.com/ibara/mg"
-  # https://devio.us/~bcallah/mg/mg-20180421.tar.gz is temporarily offline
-  url "https://dl.bintray.com/homebrew/mirror/mg-20180421.tar.gz"
-  sha256 "11215613a360cf72ff16c2b241ea4e71b4b80b2be32c62a770c1969599e663b2"
+  url "https://github.com/ibara/mg/releases/download/mg-6.5/mg-6.5.tar.gz"
+  sha256 "3e4bb4582c8d1a72fb798bc320a9eede04f41e7e72a1421193174b1a6fc43cd8"
 
```

Comment:
Can you add the following, it'll reset the numbering
```suggestion
version_scheme 1
```

Reply:
Done